client/rest-high-level/src/test/java/org/elasticsearch/client/GetAliasesResponseTests.java

```diff
@@ -0,0 +1,237 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client;
+
+import org.elasticsearch.cluster.metadata.AliasMetaData;
+import org.elasticsearch.common.xcontent.XContentParser;
+import org.elasticsearch.common.xcontent.json.JsonXContent;
+import org.elasticsearch.rest.RestStatus;
+import org.elasticsearch.test.AbstractXContentTestCase;
+import org.elasticsearch.test.EqualsHashCodeTestUtils;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class GetAliasesResponseTests extends AbstractXContentTestCase<GetAliasesResponse> {
+
+    @Override
+    protected GetAliasesResponse createTestInstance() {
+        RestStatus status = randomFrom(RestStatus.OK, RestStatus.NOT_FOUND);
+        String errorMessage = RestStatus.OK == status ? null : randomAlphaOfLengthBetween(5, 10);
+        return new GetAliasesResponse(status, errorMessage, createIndicesAliasesMap(0, 5));
+    }
+
+    private static Map<String, Set<AliasMetaData>> createIndicesAliasesMap(int min, int max) {
```

Comment:
Maybe reuse the static utils from `org.elasticsearch.action.admin.indices.alias.get.GetAliasesResponseTests` ?

Reply:
tried that, but I went for simple maps while the internal class uses ImmutableOpenMap

Reply:
:facepalm: Missed that. 
What about the `createAliasMetaData` and `mutateAliasMetaData`

Reply:
I think that I tried that too but we would need to add a test dependency to get that and I didn't feel like we need it so badly.